backend/geneaprove/sql/personas.py

```diff
@@ -14,6 +15,10 @@
 logger = logging.getLogger(__name__)
 
 
+# Identify database backend
+database_in_use = settings.DATABASES['default']['ENGINE'].split('.')[-1]
```

Comment:
You actually do not need the 'split', since below you use `in` operator which will work just as well if you don't split

Reply:
Since we no have other packages similar to personas.py, perhaps those database-specific handling could be moved to `sqlset.py`, so that it is available to all those low-level providers (at least that had been my intention behind creating sqlset.py).

Reply:
I'd intended to do just that (clean up the database check), but you beat me to it! :D Nicely done, btw. And yes, moving them to `sqlset` makes sense. Perhaps more of those would fit there well.